src/components/Auth.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
```

Comment:
Auth.js can be placed inside a separate folder call modules.

Components folder should contain shared reuseable components

Reply:
Okay